lib/windows.cc

```diff
@@ -9,36 +9,42 @@
 typedef int(__stdcall *lp_GetScaleFactorForMonitor)(HMONITOR,
                                                     DEVICE_SCALE_FACTOR *);
 
-struct Process {
+struct Process
+{
```

Comment:
I would rather like to keep `{` in the same line.

Reply:
it just syntax fixer preference